packages/wonder-stuff-sentry/src/sentry-data-reducer.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import type {SentryData} from "./types.js";
+
+/**
+ * Reduce the values of one sentry data object into another.
+ *
+ * For `tags` and `contexts`, existing values are overwritten.
+ * For `fingerprints`, the arrays are concatenated together.
+ *
+ * This method works best with `reduce` or `reduceRight`.
+ *
+ * @param {SentryData} accumulator The target object being reduced into.
+ * @param {?$ReadOnly<SentryData>} current The next SentryData object to be reduced.
+ * @param {number} index The index in the array being reduced of the current
+ * object being reduced.
+ * @param {$ReadOnlyArray<SentryData>} array The array being reduced.
+ * @returns {SentryData} The reduced sentry data.
+ */
+export const sentryDataReducer = (
+    accumulator: SentryData,
+    current: ?$ReadOnly<SentryData>,
+    index: number,
+    array: $ReadOnlyArray<?SentryData>,
+): SentryData => {
+    if (current == null) {
+        return accumulator;
+    }
```

Comment:
You're programming style is very defensive.  I could definitely stand to adopt some of these practices. 🙂

Reply:
What else can we add to this? Jeff's programming style is very:
- defensive
- clear
- well-architected and encapsulated
- polished and well documented
- consistent with itself (← sooooo underrated!)
- easy to understand
- thoroughly tested

It's like all the #programminggoals!